pipeline/recon-pipeline.py

```diff
@@ -77,6 +76,8 @@ def cluge_package_imports(name, package):
     port_results_parser,
 )
 
+from .tools import tools
```

Comment:
`from .tools import tools  # noqa: F401,E402`

adding the comment at the end will suppress the flake8 warnings.  The imports are after logical code to allow for an improved command line user experience.  We do this with forethought and malice, and accept that we are breaking flake8 recommendations in doing so.